website/source/docs/auth/github.html.md

```diff
@@ -68,4 +72,15 @@ Success! Data written to: auth/github/map/teams/owners
 ```
 
 The above would make anyone in the "owners" team a root user in Vault
-(not recommended).
+(not recommended). Note: the team ("owners") is case-sensitive.
```

Comment:
We fixed this in https://github.com/hashicorp/vault/pull/184, it just hasn't been released yet.